lib/vector-buffers/src/buffer_usage_data.rs

```diff
@@ -78,12 +78,17 @@ impl BufferUsageHandle {
             .fetch_add(byte_size, Ordering::Relaxed);
     }
 
-    /// Attempts to increment the count of dropped events for this buffer component.
+    /// Attempts to increment the number of dropped events (and their total size) for this buffer component.
     ///
     /// If the component itself is not configured to drop events, this call does nothing.
-    pub fn try_increment_dropped_event_count(&self, count: u64) {
-        if let Some(dropped_event_count) = &self.state.dropped_event_count {
-            dropped_event_count.fetch_add(count, Ordering::Relaxed);
+    pub fn try_increment_dropped_event_count_and_byte_size(&self, count: u64, byte_size: u64) {
```

Comment:
`try` methods usually return `Result` or `Option`. Does this naming make sense here?

Reply:
I think maybe it's called `try_` since it may or may not actually increment depending on whether `self.state.dropped_event_count` is `Some` or not. I'm not sure if there is a better naming pattern for that though.